ui/app/src/components/WorkflowCancellationDialog/WorkflowCancellationDialogContainer.tsx

```diff
@@ -37,48 +36,30 @@ const useStyles = makeStyles()((theme) => ({
 }));
 
 export function WorkflowCancellationDialogContainer(props: WorkflowCancellationDialogContainerProps) {
-  const { items, onClose, onContinue } = props;
+  const { items = [], onClose, onContinue } = props;
```

Comment:
Remove default and if items is nullish, render an empty state with "No items were provided as input"